File: Java/프로그래머스/0/120819. 아이스 아메리카노/아이스 아메리카노.java
class Solution {
    public int[] solution(int money) {
        int change = 0;
        int n = 0;
        int i=1;
        while(true) {
        if(money-(i*5500)<0) {
            change = (money-(i-1)*5500);
            n=(i-1); 
            break;
            }
        else i++;
        }
        int[] answer = {n,change};
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120819. 아이스 아메리카노/아이스 아메리카노.js
function solution(money) {
    var answer = new Array(2);
    var i=1;
    while(true){
        if(5500*i>money){
            i= i-1;
            break;
        }
        i++;
    }
    answer[0]=i;
    answer[1]=money-5500*i;
    return answer;
}